tests/common/factories/job.py

```diff
@@ -0,0 +1,14 @@
+from factory import Faker
+
+from h import models
+
+from .base import ModelFactory
+
+
+class Job(ModelFactory):
+    class Meta:
+        model = models.Job
+
+    name = "test_job"
+    priority = Faker("random_element", elements=[1, 100, 1000, 10000])
+    tag = "test_tag"
```

Comment:
I wish I could use LMS's much nicer style for creating factories but that's probably a can of worms -- h has a weird setup for factories -- so I've just followed the existing style for h factories